install/installer/pkg/components/public-api-server/configmap.go

```diff
@@ -115,6 +122,38 @@ func getStripeConfig(cfg *experimental.Config) (corev1.Volume, corev1.VolumeMoun
 	return volume, mount, path, true
 }
 
+func getOIDCClientJWTSecretConfig(cfg *experimental.Config) (corev1.Volume, corev1.VolumeMount, string, bool) {
+	var volume corev1.Volume
+	var mount corev1.VolumeMount
+	var path string
+
+	if cfg == nil || cfg.WebApp == nil || cfg.WebApp.PublicAPI == nil || cfg.WebApp.PublicAPI.OIDCClientJWTSigningKeySecretName == "" {
+		return volume, mount, path, false
+	}
+
+	oidcClientJWTSigningKeySecretName := cfg.WebApp.PublicAPI.OIDCClientJWTSigningKeySecretName
+	path = oidcClientJWTSigningKeyMountPath
+
+	volume = corev1.Volume{
+		Name: "oidc-client-jwt-signing-key",
+		VolumeSource: corev1.VolumeSource{
+			Secret: &corev1.SecretVolumeSource{
+				SecretName: oidcClientJWTSigningKeySecretName,
+				Optional:   pointer.Bool(true),
```

Comment:
We'll probs want to remove this, once we have it as a hard dependency.

Reply:
Thanks for mentioning. I think we need it to roll out smoothly, but then make it a hard dependency to fail fast.